.mergify.yml

```diff
@@ -31,8 +31,21 @@ pull_request_rules:
       - base=main
       - files~=^code\/parachain\/runtime\/picasso\/
       - branch-protection-review-decision=APPROVED
-
     actions:
       queue:
         priority: high
+        name: default
+
+  # Reduce reviewers for subsquid since QA tests it vigorously anyway
```

Comment:
I don't entirely agree with this statement. Code review is not just to catch bugs. It involves many other aspects such as design review for new features as well.

Reply:
I agree with your statement however not with the reality of the validity of code reviews atm.
I spent multiple hours 2 days ago trying to figure out bugs on production and I've never seen any substantial changes suggested to subsquid related PRs so far.
If you believe there are aspects that trump whatever other priority you have to review, please go ahead, requesting changes should block the PR from being merged

Reply:
>I spent multiple hours 2 days ago trying to figure out bugs on production and I've never seen any substantial changes suggested to subsquid related PRs so far.

This is not enough sample size to evaluate the review system for Subsquid I can tell you that.